cloudinit/config/cc_growpart.py

```diff
@@ -316,6 +369,23 @@ def resize_devices(resizer, devices):
                          "failed to resize: disk=%s, ptnum=%s: %s" %
                          (disk, ptnum, e),))
 
+        if is_lvm and isinstance(resizer, ResizeGrowPart):
+            try:
+                if len(devices) == 1:
+                    (_out, _err) = subp.subp(["lvm", "lvextend", "--extents=",
+                                             "100%FREE", blockdev],
```

Comment:
this is probably wrong, or at best looks wierd.

you'll execute:

    lvm lvextend --extents= 100%FREE <blockdev>

Even if that works, what we *want* to have is '--extents=100%FREE' as one argument not two.

    lvm lvextend --extents=100%FREE <blockdev>

I realize pycodestyle is causing you pain here. this should work:

```python

                   (_out, _err) = subp.subp(
                        ["lvm", "lvextend", "--extents=100%FREE", blockdev],
                        update_env={'LANG': 'C'})
```

Reply:
I was having code style problems because I couldn't run tox on my laptop because of the issue of PR #791 . But now I can and everything is better :-)